### ISSUE TITLE
RFC: change default of group setting storage to none in Paket 6

### ISSUE THREAD

**Issue by forki** [fsprojects/Paket#3746]
I propose we change the default of the `storage` setting to `none`. Currently the default is set to `packages` but since more and more users convert to SDK style projects that switch feels wrong.

More details: https://fsprojects.github.io/Paket/dependencies-file.html#Disable-packages-folder

**Comment by slang25** [comment]
Yes 😃 This is the first thing I change in most projects, that and the `framework` to `auto-detect`

**Comment by matthid** [comment]
Does the `framework` option even have a meaning in new SDK projects?

**Comment by enricosada** [comment]
👍 for `storage: none`

> Does the framework option even have a meaning in new SDK projects?
@matthid i use it to constraint the resolution when is a multi targeted framework. i am doing it wrong?

**Comment by BlythMeister** [comment]
Yes, 100% and framework auto detect makes package resolution so much faster!

Why resolve all transatives for every framework....ain't nobody got time for that.

**Comment by isaacabraham** [comment]
To be fair the new init settings have packages none and versions to netstandard2 and netcore3, right?

**Comment by matthid** [comment]
I mean this is all off topic but isn't it more painful for new users? Yes it is faster but if I add a new proejct with a new framework the lockfile will not be updated and I might run into issues. How do I know that I have to run `paket update` (`paket install` probably won't even work as paket will say every group is up to date and nothing has changed in the depsfile)

**Comment by forki** [comment]
I'm not proposing to add autodetect.

**Comment by matthid** [comment]
Well, I'm not 100% sure the above is correct, just an assumption which should be tested. Otherwise I think changing the default is not a bad idea if that scenario "just works".
